### DISCUSSION TITLE
Table virtual massive data

### DISCUSSION THREAD

**Discussion by aboucodeur** [mantinedev/mantine#3383] [Feedback]
Hello i'm using mantin 1 years ago . The probleme is i need virtualized massi data but is slow to rendering

**Comment by rtivital** [comment]
Mantine does not provide any components for virtualization. You should use one of the specialized libraries for this.

**Comment by aboucodeur** [comment]
Thanks for your response .You can add this is new version , mantine is my main library to build user interface eg:Stock Management application

**Reply by rtivital** [reply]
Nope, it is not planned to be added any time soon.

**Comment by aboucodeur** [comment]
Thanks, it's will be my contribution in mantine ui my favorite ui library in react.
But today in my country Mali mantine ui level up my application thanks @rtivital ...
